examples/quest-1-simple/whisperer.yaml

```diff
@@ -0,0 +1,17 @@
+entity_type: Quest
+schema_version: 1
+
+# Quest Attributes
+id: my-awesome-quest
+
+tags: [sample, life-changing, gcp]
+logo: ./quest-logo.png
+default_locale: en
+
+steps:
+  - content: lab-minimal
```

Comment:
Do you think this will uniquely identify any piece of content (in the bundle) or would we have to include a `type: {lab|file|url}` as well?

Reply:
I think it's actually identifying content in the same library as where this quest bundle is pulled from... which for the purposes of git-whisperer means a folder name in the `labs` folder.